_adr/81/index.adoc

```diff
@@ -19,10 +19,243 @@ NOTE: https://datatracker.ietf.org/doc/html/rfc2119[RFC-2119] keywords are used
 
 // No need for a title heading, it's added by the template
 
-## Context and Problem Statement
+== Context and Problem Statement
 // What is the background against which this decision is being taken?
 
 Excluding storage sizing, the Kafka service currently supports a single size of Kafka cluster composed of 3 broker pods with fixed CPU, memory and density. 
 In order for the service to be more broadly applicable to production Kafka use cases we need to be able to provide it at a variety of scales.
 This means that when a user creates a Kafka instance they get to specify "how big" the instance they're creating should be. 
 
+== Goals
+// Bulleted list of outcomes that this ADR, if accepted, should help achieve
+
+* Define how instance scaling is modelled conceptually and in the interfaces between the UI, fleet manager and fleet shard.
+* Avoid tying our hands to any particular unit of scaling.
+* Enable other profiles of the service in the future, without having to change the contracts between components.
+
+== Non-goals
+// Bulleted list of outcomes that this ADR is not trying to achieve.
+
+* Consistency of scaling across different infrastructures (both on the same cloud, and between different clouds).
+* Define how the fleet manager manages shard capacity.
+* Storage scaling, which already supported in the fleet manager and shard, and is orthogonal to the scaling of clusters under consideration here.
+
+== Current situation
+// Where are we now?
+
+The service does not offer any scaling options, except for storage, which is orthogonal to this problem.
+
+== Proposal
+// What is the decision being proposed
+
+=== Conceptual model
+Conceptually we define a _series_, which is an ordered list of _sizes_ of instance. 
+
+A _series_ has a name, and a description. The description might include what is common across the sizes it supports (e.g. "3 AZ").
```

Comment:
I'm not sure "series" is the most intuitive name here - it took me minutes to realize what it was.

Reply:
I'm not wedded to it, but any suggestions?

Reply:
T-shirts! /Me run away.

Reply:
- Streaming Unit Profiles
- Capacity Profiles

Reply:
I'd much prefer a single word, and preferably a short one. But "profile" instead of series is fine imo?